tests/cli/test_rasa_x.py

```diff
@@ -80,6 +85,41 @@ def test_if_endpoint_config_is_invalid_in_local_mode(kwargs: Dict):
     assert not x._is_correct_event_broker(config)
 
 
+def test_overwrite_for_local_x(caplog: LogCaptureFixture):
```

Comment:
I think we should split up the test because it's testing multiple things:
- is the model url overwritten (e.g. `test_overwrite_model_server_url`)
- is the `wait_between_pull_time` overwritten (`test_reuse_ wait_time_between_pulls`)


Also we should add tests for the case
- no `wait_between_pull_time` was specified - is the default value correctly set
- no model server url was set - is the default value correctly set?

Reply:
ok I'll work on refactoring this out, thanks

Reply:
these should all be there now in the latest commit